utils/server-side-helper/app/get-url.ts

```diff
@@ -0,0 +1,7 @@
+export const getURL = (path: string = '') => {
```

Comment:
getUrl is very generic. A more self explaining function name and a few comments will improve readability 

like `getAbsoluteSiteUrl()`

Reply:
I renamed to getAbsoluteSiteUrl, good idea !